disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -480,6 +480,35 @@ func (s *BlobMetadataStore) GetDispersalResponse(ctx context.Context, batchHeade
 	return res, nil
 }
 
+func (s *BlobMetadataStore) GetDispersalResponses(ctx context.Context, batchHeaderHash [32]byte) ([]*corev2.DispersalResponse, error) {
+	items, err := s.dynamoDBClient.Query(ctx, s.tableName, "PK = :pk AND begins_with(SK, :prefix)", commondynamodb.ExpressionValues{
```

Comment:
Do you think we will need pagination? Or is the operator set small enough not to worry about it?

Reply:
It should be small enough to worry about

Reply:
One optimization we may do in the future is to denormalize and just store all operators' responses for a batch in a single message (and then this request will just need a lookup, no need for index)

Reply:
I wonder if that could hit size limit on DynamoDB for a record

Reply:
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ServiceQuotas.html, says around 400kKB

Reply:
That's good angle. It should be sufficient I think: the response size is about 160 bytes (if no error message, which can be a variable length), which is about 2500 operators. We need to check it again if we take this approach (e.g. maybe we add more info to the response).